app/views/catalog/_show_top_fields.html.erb

```diff
@@ -3,8 +3,14 @@
 <dl class="row">
   <% blacklight_config.show_fields.each do |field_name, field| -%>
     <% if field.top_field && !document[field_name].nil? %>
-        <dt class="blacklight-<%= field_name.parameterize %>-label col-md-3"><%= render_document_show_field_label document, field: field_name %></dt>
-        <dd class="blacklight-<%= field_name.parameterize %> col-md-9"><%= doc_presenter.field_value field %></dd>
+      <dt class="blacklight-<%= field_name.parameterize %>-label col-md-3
+          <%= 'text-muted' unless field[:sr_only] -%>
+          <%= 'sr-only' if field[:sr_only] -%>">
+        <%= render_document_show_field_label document, field: field_name %>
+      </dt>
+      <dd class="blacklight-<%= field_name.parameterize %> col-md-9"><%= doc_presenter.field_value field %></dd>
```

Comment:
We don't have any `sr_only` fields as show fields, but, if we do in the future we'll have this all set for it.